src/app/clusters/tls-certificate-management-server/CertificateTableImpl.h

```diff
@@ -0,0 +1,94 @@
+/**
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+#include <app/clusters/tls-certificate-management-server/CertificateTable.h>
+#include <app/storage/FabricTableImpl.h>
+#include <app/util/attribute-storage.h>
+#include <app/util/config.h>
+#include <lib/core/DataModelTypes.h>
+#include <lib/support/Pool.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace Tls {
+
+#if defined(TLS_CLIENT_CERTIFICATE_SIZE_PER_FABRIC) && TLS_CLIENT_CERTIFICATE_SIZE_PER_FABRIC
```

Comment:
Usually this sort of config thing would get defaulted in a config header, so that over here we would just use its value and if we are not including the right config header it would fail to compile instead of silently doing the default.